azure/functions/decorators/function_app.py

```diff
@@ -188,6 +214,10 @@ def add_binding(self, binding: Binding) -> 'FunctionBuilder':
         self._function.add_binding(binding=binding)
         return self
 
+    def add_setting(self, setting: Setting) -> 'FunctionBuilder':
```

Comment:
Should this be a private method?

Reply:
Nope. It needs to be accessed by other classes. Its used same as add_trigger and add_bindings